content/applications/sales/crm/track_leads/lead_scoring.rst

```diff
@@ -1,73 +1,163 @@
-=============================
-Assign leads based on scoring
-=============================
+=========================================
+Assign leads with predictive lead scoring
+=========================================
 
-*Lead Scoring* lets you rank and prioritize leads based on selected criteria. A higher score
-indicates that a lead is more likely to bring value to your company.
+The *Odoo CRM* app can automatically assign leads/opportunities to sales teams and salespeople. A
+standard practice is to assign leads based on the probability of winning each lead. Companies can
+prioritize the leads that are more likely to result in successful deals by quickly assigning them
+to the appropriate salespeople.
 
-For example, if customers from Canada tend to be more likely to purchase your products, you can
-assign them a higher score than customers from other countries.
+Odoo automatically calculates the probability of winning each lead using a method called *predictive
+lead scoring*.
 
-Configuration
-=============
+Predictive lead scoring
+=======================
+
+Predictive lead scoring is a machine-learning model that uses historical data from *Odoo CRM* to
+score open leads/opportunities. As a company processes opportunities through the CRM pipeline, Odoo
+collects data on which opportunities are won and lost. Predictive lead scoring uses this data to
+predict the probability of winning each new lead or opportunity. The more opportunities that are
+sent through the CRM pipeline, the more data Odoo collects, and the more accurate the probabilities
+become.
```

Comment:
...score open leads/opportunities. 

As a company processes opportunities through the CRM pipeline, Odoo
collects data on which opportunities are won and lost. Predictive lead scoring uses this data to
predict the probability of winning each new lead or opportunity. 

The more opportunities that are sent through the CRM pipeline, the more data Odoo collects, resulting in more accurate probabilities.